pkg/csi/service/node.go

```diff
@@ -641,10 +647,13 @@ func (driver *vsphereCSIDriver) NodeGetInfo(
 
 	var nodeInfoResponse *csi.NodeGetInfoResponse
 
-	nodeID := os.Getenv("NODE_NAME")
-	if nodeID == "" {
+	nodeName := os.Getenv("NODE_NAME")
+	if nodeName == "" {
 		return nil, logger.LogNewErrorCode(log, codes.Internal, "ENV NODE_NAME is not set")
 	}
+	// NOTE: In future, we plan to use NodeVM UUID as NodeID.
```

Comment:
Can you change this to a `TODO` with explanation of why this needs to be changed?

Reply:
It is a separate feature in itself but I mentioned it here because I am reassigning the value of `nodeName` to another variable `nodeID`. In future, when we assign NodeVM UUID to `nodeID` variable, we do not need to change the rest of the code. Just changing this line will suffice. I can remove the TODO if it is causing any confusion.

Reply:
Removed the TODO